autoware_api_launch/launch/autoware_api.launch.xml

```diff
@@ -2,6 +2,7 @@
 
   <arg name="respawn" default="true" />
   <arg name="launch_default_ad_api" default="true"/>
+  <arg name="max_message_size" default="100000000" />
```

Comment:
Can we use `Infinity`? :thinking: 
https://github.com/RobotWebTools/rosbridge_suite/commit/0751df37634966e63cedc8f32178a62c37329d1c

Reply:
But I guess we should avoid using `rosbridge` and use `rclpy` etc. if we send such large data.

Reply:
＞ But I guess we should avoid using rosbridge and use rclpy etc. if we send such large data.

I think so too.
It is better that the permanent measure should be to split and load large data like map.

Reply:
I check to use `Infinity` . For don't change it again.

Reply:
This parameter cannot be set "Infinity"(integer only)
So I changed this parameter to 1000*1000*1000 because PCD can be over 100 MB.

```
rclpy.exceptions.InvalidParameterTypeException: Trying to set parameter 'max_message_size' to 'inf' of type 'DOUBLE', expecting type 'INTEGER'
```